game/domain/sector.lua

```diff
@@ -134,16 +131,9 @@ function Sector:makeTiles(grid)
     for j = 1, self.w do
       local tile = false
       local tile_type = grid.get(j, i)
-      if grid.get(j, i) == SCHEMATICS.FLOOR then
-        if (i+j) % 2 == 0 then
-          tile = { unpack(COLORS.FLOOR1) }
-        else
-          tile = { unpack(COLORS.FLOOR2) }
-        end
-      elseif grid.get(j, i) == SCHEMATICS.EXIT then
-        tile = { unpack(COLORS.EXIT) }
+      if tile_type and tile_type ~= SCHEMATICS.NAUGHT then
+        tile = { type = tile_type }
```

Comment:
Are you sure it saves as `' '`? Here it transforms into false, and then when you save it will stay false.

Reply:
Oh, right. I meant to say tiles didn't use just 3 characters, but indeed the `NAUGHT` tile is still saved as `false`. Want me to treat it the same as the others?

Reply:
I think it's better to treat it the same if we're not going to have any particular use for the boolean value. The sectorview code, however, does become shorter with the false value. At this point in the development I don't think there is a difference.

Reply:
I just thought that the char value is actually "hasheable" meanwhile false isn't. So there's that.

Reply:
I tried to change it to follow the other types of tiles, but realized there's actually a number of places in the code that assume false tiles mean empty tiles. I'll open an issue and leave it for later.